### ISSUE TITLE
Documentation Update: With the introduction of "developer" and "kubeadm" roles switching to a cluster admin / kubeadmin a pre-requisite to list operators

### ISSUE THREAD

**Issue by rohanarora** [code-ready/crc#1989]
**URL**: https://access.redhat.com/documentation/en-us/red_hat_codeready_containers/1.22/html/getting_started_guide/using-codeready-containers_gsg#accessing-the-openshift-cluster-with-oc_gsg
**Step 4 of Section: 3.3.2: Accessing the OpenShift cluster with oc**

### Currently
```
oc get co
```
results in
```
Error from server (Forbidden): clusteroperators.config.openshift.io is forbidden: User "developer" cannot list resource "clusteroperators" in API group "config.openshift.io" at the cluster scope
```

### Could be updated with
With the introduction of of "developer" and "kubeadm" roles switching to the cluster admin role is a pre-requisite to list OpenShift Cluster Operators. You could do so by:
```
oc login -u kubeadmin -p password https://api.crc.testing:6443
```
followed by 
You can now use oc to interact with your OpenShift cluster. For example, to verify that the OpenShift cluster Operators are available:
```
oc get co
```

**Comment by rohanarora** [comment]
More than happy to make a PR for this @praveenkumar @robin-owen. Let me know.

**Comment by guillaumerose** [comment]
I am wondering if we need 2 accounts. We could set the developer to be "cluster-admin" and remove kubeadmin account. 
I found the current developer account to be useless/too restrictive. Almost every oc command fail with it.

See https://docs.openshift.com/container-platform/4.6/post_installation_configuration/preparing-for-users.html#creating-cluster-admin_post-install-preparing-for-users

**Comment by cfergeau** [comment]
We making assumptions on what role/permissions a developer would get on a "real" cluster. Unlikely they'd be given an admin role, and using an account with restrictive permissions potentially avoids some unexpected permission issues when moving from crc to a production env.

**Comment by guillaumerose** [comment]
Could it be something we document ? Having one account simplify the start message. Also, I found the console really deceptive when I open it as developer: everything is empty.

**Comment by cfergeau** [comment]
There was an issue about creating a default project, which would avoid the empty console I think? I'm not really in favour of that though :) I'm wondering if the responsibility to create a 'proper' user, project, ... should go to odo/vscode/...?

**Comment by robin-owen** [comment]
Thank you for the report, @rohanarora!

Early on, the `oc get co` command worked properly from the `developer` account -- I'm actually not sure at what point that changed.

Part of the problem here is that from a docs perspective, we don't want to directly advocate that users access the cluster with administrative permissions unless necessary to perform administrative actions.This means that we don't really want to recommend logging in as `kubeadmin` in documentation meant as an introduction for accessing the cluster via `oc`.

Is there an alternative `oc` command that we can recommend users run in order to verify that they can access the cluster with `developer` permissions? All we need is a verification step to document without requiring admin (it's up to the user, of course, to later decide which account to use).

**Comment by rohanarora** [comment]
Sure thing, @robin-owen. Dug a bit more into this.

### Probable Cause
From [1] the developer account is being granted sudoer privileges
```
retry ${OC} --kubeconfig $1/auth/kubeconfig create clusterrolebinding developer --clusterrole=sudoer --user=developer
```
Wondering here if the change seems to to be the introduction of [2].

### Tentative Modification(s) / Addition(s) 
From a documentation perspective we could propose switching to
```
oc get co --as system:admin
```
This way the user is *aware* in a way that they are performing an *administrative* action which is / was the intend behind introduction of the updated(?) impersonation system as well.

```
➜ ✗ oc get co
Error from server (Forbidden): clusteroperators.config.openshift.io is forbidden: User "developer" cannot list resource "clusteroperators" in API group "config.openshift.io" at the cluster scope

➜  ✗ oc get co --as system:admin
NAME                                       VERSION   AVAILABLE   PROGRESSING   DEGRADED   SINCE
authentication                             4.6.15    True        False         False      4m8s
cloud-credential                           4.6.15    True        False         False      13d
....
....
...
...
```

In addition to the above the mechanism for the user to validate could be to check if they belong to the role ClusterRole/admin (?). Based on [3]
```
➜  ✗ oc get rolebindings
NAME                    ROLE                               AGE
admin                   ClusterRole/admin                  3d16h
system:deployers        ClusterRole/system:deployer        3d16h
system:image-builders   ClusterRole/system:image-builder   3d16h
system:image-pullers    ClusterRole/system:image-puller    3d16h
```

[1]: https://github.com/code-ready/snc/blob/ab91001c6bd8f5e6d5c207459fd6241fe4540cde/createdisk.sh#L42
[2]: https://docs.openshift.com/container-platform/4.6/authentication/impersonating-system-admin.html#impersonation-system-admin-group_impersonating-system-admin
[3]: https://docs.openshift.com/container-platform/3.3/admin_solutions/user_role_mgmt.html#view-roles-users-cluster

**Comment by robin-owen** [comment]
For now, I think we should go with `oc get co --as system:admin` while explicitly noting that this is an administrative action. Writing up a PR now. Thank you!

**Comment by robin-owen** [comment]
Fixed by #2021.
